test/automation/src/code.ts

```diff
@@ -188,7 +188,7 @@ export class Code {
 			let done = false;
 
 			// Start the exit flow via driver
-			this.driver.exitApplication().then(veto => {
+			this.driver.exitApplication().then((veto: any) => {
```

Comment:
Was this an accident? Don't think you touch veto in the changes

Reply:
My VS Code was actually giving me an error on that symbol due to the implicit `any`.